cogito/putter.go

```diff
@@ -134,22 +135,40 @@ func (putter *ProdPutter) ProcessInputDir() error {
 	return nil
 }
 
-func (putter *ProdPutter) Sinks() []Sinker {
-	return []Sinker{
-		GitHubCommitStatusSink{
+func (putter *ProdPutter) Sinks() ([]Sinker, error) {
+	var err error
+	supportedSinks := map[string]Sinker{
+		"github": GitHubCommitStatusSink{
 			Log:     putter.log.Named("ghCommitStatus"),
 			GhAPI:   putter.ghAPI,
 			GitRef:  putter.gitRef,
 			Request: putter.Request,
 		},
-		GoogleChatSink{
+		"gchat": GoogleChatSink{
 			Log: putter.log.Named("gChat"),
 			// TODO putter.InputDir itself should be of type fs.FS.
 			InputDir: os.DirFS(putter.InputDir),
 			GitRef:   putter.gitRef,
 			Request:  putter.Request,
 		},
 	}
+	sinksParams := putter.Request.Params.Sinks
+	if len(sinksParams) == 0 {
+		// No sink specified, it will return all supported sinks.
+		sinksParams = make([]string, 0, len(supportedSinks))
+		for k := range supportedSinks {
+			sinksParams = append(sinksParams, k)
+		}
```

Comment:
While this is currently true, it might not stay as such when adding another sink. In my understanding, the current behaviour is only here to provide backward compatibility. So this should be changed to something like:
```suggestion
		// No sink specified, we default to github and ghcat for backward compatibility.
		sinksParams = []string{"github", "gchat"}
```